styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
```

Comment:
[Optional] Kindly provide some species such as word-break, padding, font styles, etc. for other elements such as text inputs, that would make the page more well-looking. :relaxed: